kogito-business-rules-quarkus/README.md

```diff
@@ -68,8 +98,26 @@ To make use of this application it is as simple as putting a sending request to
 
```

Comment:
(line 94) [Submit a request] missing a ':' in content shown:

```
{
  "person" : {
    "name": "john",
    "age" : 20
  }
}
```

Reply:
Fixed @radtriste